### PR TITLE
Rewrite POST requests as GET requests

### PR DESCRIPTION
**Pull request by fabxc**

### CODE REVIEW

File: cmd/frontend/main.go [hunk 1]
```diff
@@ -154,9 +154,12 @@ func forward(logger log.Logger, target *url.URL, transport http.RoundTripper) ht
 	return http.HandlerFunc(func(w http.ResponseWriter, req *http.Request) {
 		u := *target
 		u.Path = path.Join(u.Path, req.URL.Path)
-		u.RawQuery = req.URL.RawQuery
 
-		newReq, err := http.NewRequestWithContext(req.Context(), req.Method, u.String(), req.Body)
+		// Write all params into the URL and make a GET request.
```
Comment (pintohutch) [inline cmd/frontend/main.go:158]: Can we add a little more context for why in the comment?

**Review by pintohutch — COMMENTED** [review]
Did you test this?

File: cmd/frontend/main.go (hunk 1 above)
Comment (fabxc) [reply cmd/frontend/main.go]: Updated. Also constrained it to that one path just in case.

**Comment by fabxc** [comment]
Yes

**Review by pintohutch — APPROVED** [review]

**Pull request merged**